modules/sdk-coin-tia/test/integration/index.ts

```diff
@@ -0,0 +1,16 @@
+import 'should';
+
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { TestBitGo, TestBitGoAPI } from '@bitgo/sdk-test';
+
+import { Tia, Ttia } from '../../src/index';
+
+describe('TIA', function () {
+  let bitgo: TestBitGoAPI;
+  before(function () {
+    bitgo = TestBitGo.decorate(BitGoAPI, { env: 'mock' });
+    bitgo.safeRegister('tia', Tia.createInstance);
+    bitgo.safeRegister('ttia', Ttia.createInstance);
+    bitgo.initializeTestVars();
+  });
+});
```

Comment:
Seems like a duplicate of what's already there in `tia.ts` unit test file. We can delete this file unless this has been added here for a follow up PR